OOP/News.py

```diff
@@ -0,0 +1,90 @@
+# Создайте инструмент, который будет создавать новостную ленту, сгенерированную пользователем:
+# 1. Пользователь выбирает, какой тип данных он хочет добавить
+# 2. Предоставьте требуемые данные типа записи
+# 3.Запись публикуется в текстовом файле в специальном формате
+
+# Вам необходимо реализовать:
+# 1.Новости – текст и город в качестве ввода. Дата рассчитывается во время публикации.
+# 2.Приватное объявление – на входе текст и дата истечения срока действия. Оставшийся день рассчитывается во время публикации.
+# 3. Ваш уникальный с уникальными правилами публикации.
+
```

Comment:
Please no Russian in the code, task is in English, why did you translate it?

Reply:
fixed